config/GPVE01/config.yml

```diff
@@ -6,5 +6,5 @@ symbols: config/GPVE01/symbols.txt
 splits: config/GPVE01/splits.txt
 
 mw_comment_version: 10 # GC Linker 2.6
-symbols_known: true # Very fast analysis
+symbols_known: false # Very fast analysis
```

Comment:
Similarly, this change is just to match the upstream repo.

If setting this to `true` is important, please let me know.